src/contracts/core/PaymentCoordinatorStorage.sol

```diff
@@ -30,6 +30,10 @@ abstract contract PaymentCoordinatorStorage is IPaymentCoordinator {
     uint32 public immutable MAX_FUTURE_LENGTH;
     /// @notice absolute min timestamp (seconds) that a payment can start at
     uint32 public immutable GENESIS_PAYMENT_TIMESTAMP;
+    /// @notice The cadence at which a snapshot is taken offchain for calculating payment distributions
+    uint32 public constant SNAPSHOT_CADENCE = 1 days;   
```

Comment:
Do we need the cadence to be stored in the contract? It's not being used in the contract and I'm not sure if we ever need to RPC call this offchain.

Reply:
We don't use RPC to call this off chain, but it should be the case that `CALCULATION_INTERVAL_SECONDS` is a multiple of this, which is why I added it